scripts/build-stories.js

```diff
@@ -119,8 +119,9 @@ async function buildStories() {
       const story = await readFile(outFile, 'utf8');
       await writeFile(outFile, buildStoryMeta(story, meta), 'utf8');
     } catch (e) {
-      console.error(e);
-      process.exit(-1);
+      // console.error(e);
```

Comment:
This will be fixed in a separate branch. Please, remove it from here.

Reply:
done